pennylane_qiskit/qiskit_device2.py

```diff
@@ -208,6 +208,61 @@ class QiskitDevice2(Device):
             with those in kwargs, the settings in `options` will take precedence over kwargs. Keyword
             arguments accepted by both the transpiler and at runtime (e.g. ``optimization_level``)
             will be passed to the transpiler rather than to the Primitive.
+
+    **Example:**
```

Comment:
I don't see this `class` (and hence this code block) in the built docs. Am I missing something?

Reply:
Oh. Hmm, think I need to update the .rst file.

Reply:
If the users are not supposed to use QiskitDevice2 in any scenario, then it can stay hidden as is.